trade-finance-manager-ui/server/controllers/helpers/get-success-banner-message.helper.ts

```diff
@@ -4,7 +4,7 @@ import api from '../../api';
 import { isDealCancellationEnabled } from './deal-cancellation-enabled.helper';
 
 /**
- * Returns deal cancellation message if the deal is scheduled for cancellation
+ * Returns deal cancellation message if the deal is pending cancellation
```

Comment:
I've left this function the same name since it returns a message about the scheduled date